### DISCUSSION TITLE
How to enlarge the UI fonts?

### DISCUSSION THREAD

**Discussion by ghost** [fredvs/ideU#60] [Q&A] [answered]
It's too small on my monitor. I'm unable to read.

**Comment by fredvs** [comment, marked as answer]
You may change the UI font with menu / _Settings_ / _Extra Settings_ and change the font height like you want.

![etra-settings-ideu](https://user-images.githubusercontent.com/3421249/204161691-1714d8a0-442a-401b-9b40-38efd2efa666.png)

You may also set it with menu / _Languages_.
On the right side, choose the size of the font:

![languages-ideu](https://user-images.githubusercontent.com/3421249/204161780-0dd50c26-244b-455f-af56-d0277b5c703e.png)

**Comment by fredvs** [comment]
Since commit [e432b46](https://github.com/fredvs/ideU/commit/e432b463da44c506553b157c347822da548904b1), by default, the suggested font height for the screen resolution of the user will be used at loading.
You may disable that option in menu Extra-Settings and choose your preferred font height.

All the forms will be scaled according with the font height selected.
